couchdb/src/test/java/org/hibernate/ogm/datastore/couchdb/utils/BackendTckHelper.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.couchdb.utils;
+
+import org.junit.extensions.cpsuite.ClasspathSuite;
+import org.junit.runner.RunWith;
+
+/**
+ * Helper class allowing you to run all or any specified subset of test available on the classpath.
+ *
+ * This method is for example useful to run all or parts of the <i>backendtck</i>.
+ *
+ * @author Hardy Ferentschik
+ */
+@RunWith(ClasspathSuite.class)
+@ClasspathSuite.ClassnameFilters({ "org.hibernate.ogm.backendtck.*" })
+//@ClasspathSuite.ClassnameFilters({ ".*CRUDTest" })
+public class BackendTckHelper {
```

Comment:
Run/Debug the backendtck test within the scope of the couchdb module from within the IDE. It is just a helper. This is the alternative to copying a test from the tck into the module in order to run it.